### DISCUSSION TITLE
Specify platform for node_repositories rule

### DISCUSSION THREAD

**Discussion by dex-toshi** [bazelbuild/rules_nodejs#3040] [Q&A]
I'm running on linux, but when i try to use the node_repositories rule using either "Installation with manually specified version" or the "Installation with local vendored version" options, I get a lot of errors related to other platforms (windows, ppc, etc.) Am I doing something wrong. I don't want to build for any other platforms.

Any help would be appreciated! Thank You!

**Comment by alexeagle** [comment]
You get errors for other platforms, or info messages? The latter is expected due to the right-hand-side of https://github.com/bazelbuild/rules_nodejs/blob/stable/toolchains/node/BUILD.bazel#L96 which eagerly evaluates labels into every platform, which is annoying but should be harmless. 
I'll remove that for 5.0 since we have the correct way to reference the toolchain in genrule's now without the select.

**Reply by dex-toshi** [reply]
Hi alexeagle. That would be great if it were no longer necessary. Thank you!

**Reply by alexeagle** [reply]
Filed https://github.com/bazelbuild/rules_nodejs/issues/3124 to track it
